frontend/src/utils/platform.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * platform.js
```

Comment:
If ever you need, this file adds useful stuff for platform specific features.
 - `platform().isTouch` tells you if the platform has touch-only interactions (mobile, tablet). This can be useful to know if onMouseOver behaviors should be implemented otherwise, eg show tooltips when the user holds an element pressed rather than when the user hover the element.
 - body.className gets classes for OS & type, eg `.platform-linux .platform-desktop`; we can also add the browser if needed, eg when a browser needs specific styling corrections.